src/ui/component/datePicker/datePicker.tsx

```diff
@@ -15,8 +15,9 @@ export const DatePicker: FC<DayPickerProps> = dayPickerProps => {
   const handleSelect = useCallback(
     (date?: Date) => {
       onSelect(date)
-
-      handleOpenChange()
+      setTimeout(() => {
+        handleOpenChange()
+      }, 600)
```

Comment:
Please, be careful with timer functions, you must find a way to dispose the timer at close / unmount to avoid memory leaks.

Reply:
Sorry about that. Will update it.